### PR TITLE
GF 13-15

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

**Comment by maijapapaya** [comment]
On line 1045 in Chapter 15, I wasn't sure what to do about all the separate paragraphs. I tried some different things and nothing looked right. 👍

File: epistles-v1/original/14-epistles.adoc [hunk 1]
```diff
@@ -460,8 +461,13 @@ was before death and impatient spirits were.
 
 === Concerning the worship of God, and the Jew outward, and the inward Jew`'s offering, with their garments.
```
Comment (jaredh159) [inline epistles-v1/original/14-epistles.adoc:463]: Can you make this a `[.blurb]` -- and also the two other `=== Concerning...` headlines in this epistle?

File: epistles-v1/original/14-epistles.adoc [hunk 2]
```diff
@@ -1170,6 +1178,7 @@ which gathers to God.
 [.signed-section-signature]
```
Comment (jaredh159) [inline epistles-v1/original/14-epistles.adoc:1186]: Above here ^^ line 1083 (`So let this be read in all your men...`) -- let's make that a Postscript as well.

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Thanks, just a couple small changes, looks good.

About ch. 15 line 1045 -- I think leaving it alone was probably the right call, I'm not sure what else to do with that chunk.

File: epistles-v1/original/14-epistles.adoc (hunk 1 above)
Comment (maijapapaya) [reply epistles-v1/original/14-epistles.adoc:463]: I'm sorry about the headings, I misread Jason's message I just realized, and thought he said you did most of the headings and stuff so I didn't really bother with them thinking I would be undoing your work. 😬

Comment (jaredh159) [reply epistles-v1/original/14-epistles.adoc:463]: No need to apologize, you were right - I did basically do all of the headings, I just didn't notice these, so it was my oversight really. :smile:

**Review by jaredh159 — APPROVED** [review]
This looks great! Merging. Thanks!

**Pull request merged**